Fatima-Paola.js

```diff
@@ -0,0 +1,122 @@
+//question 1, Calculate the Cube
+
+const calculateCube = (num) => {
+    console.log(num*num*num);
+  };
+  calculateCube(5);
+  
+  
+  //question 2, Is a Vowel?
+  
+  const letters = ['a','e','i','o','u'];
+  
+   const isAVowel = (vowels) => {
+    if (letters.includes(vowels)) {
+      console.log("true");
+    } else {
+      console.log("false");
+    }
+   }
+  console.log(isAVowel("a"));
```

Comment:
nice! I would also find a way to address capital letters. It currently tells me a capital A isn't a vowel. other than that, 💯